all.tf

```
@@ -0,0 +1,100 @@
+data "aws_iam_policy_document" "all" {
+  statement {
+    sid    = "DatadogAll"
+    effect = "Allow"
+
+    actions = [
+        "apigateway:GET",
+        "autoscaling:Describe*",
+        "budgets:ViewBudget",
+        "cloudfront:GetDistributionConfig",
+        "cloudfront:ListDistributions",
+        "cloudtrail:DescribeTrails",
+        "cloudtrail:GetTrailStatus",
+        "cloudwatch:Describe*",
+        "cloudwatch:Get*",
+        "cloudwatch:List*",
+        "codedeploy:List*",
+        "codedeploy:BatchGet*",
+        "directconnect:Describe*",
+        "dynamodb:List*",
+        "dynamodb:Describe*",
+        "ec2:Describe*",
+        "ecs:Describe*",
+        "ecs:List*",
+        "elasticache:Describe*",
+        "elasticache:List*",
+        "elasticfilesystem:DescribeFileSystems",
+        "elasticfilesystem:DescribeTags",
+        "elasticloadbalancing:Describe*",
+        "elasticmapreduce:List*",
+        "elasticmapreduce:Describe*",
+        "es:ListTags",
+        "es:ListDomainNames",
+        "es:DescribeElasticsearchDomains",
+        "health:DescribeEvents",
+        "health:DescribeEventDetails",
+        "health:DescribeAffectedEntities",
+        "kinesis:List*",
+        "kinesis:Describe*",
+        "lambda:AddPermission",
+        "lambda:GetPolicy",
+        "lambda:List*",
+        "lambda:RemovePermission",
+        "logs:Get*",
+        "logs:Describe*",
+        "logs:FilterLogEvents",
+        "logs:TestMetricFilter",
+        "logs:PutSubscriptionFilter",
+        "logs:DeleteSubscriptionFilter",
+        "logs:DescribeSubscriptionFilters",
+        "rds:Describe*",
+        "rds:List*",
+        "redshift:DescribeClusters",
+        "redshift:DescribeLoggingStatus",
+        "route53:List*",
+        "s3:GetBucketLogging",
+        "s3:GetBucketLocation",
+        "s3:GetBucketNotification",
+        "s3:GetBucketTagging",
+        "s3:ListAllMyBuckets",
+        "s3:PutBucketNotification",
+        "ses:Get*",
+        "sns:List*",
+        "sns:Publish",
+        "sqs:ListQueues",
+        "support:*",
+        "tag:GetResources",
+        "tag:GetTagKeys",
+        "tag:GetTagValues",
+        "xray:BatchGetTraces",
+        "xray:GetTraceSummaries"
+    ]
+
+    resources = ["*"]
+  }
+}
+
+module "all_label" {
+  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=0.2.1"
```

```suggestion
  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=0.6.2"
```

Use the latest version